.github/workflows/main.yml

```diff
@@ -55,6 +55,7 @@ jobs:
           certificate: '${{ env.CERTIFICATE_WITHOUT_PASSWORD }}'
           folder: 'files'
           recursive: true
+          description: 'TestName'
```

Comment:
Sorry for not realizing this earlier but can you move this to the next run of the action? Those binaries are uploaded.